Update README.md (#1)
xarray-contrib/xarray-contrib: README.md

# xarray-contrib

> xarray-contrib is a GitHub organization for gathering high-quality open source project that build on [Xarray](http://xarray.pydata.org/).

Reviewer: ```suggestion
xarray-contrib is a GitHub organization for gathering high-quality open source projects that build on [Xarray](http://xarray.pydata.org/).
```

## Vision

> As more and more projects build on Xarray, we wanted to provide a central place to collect community projects. We hope that including projects under the xarray-contrib organization imporoves project visibility and increases the liklihood of sustainable open source maintance and development

Reviewer: ```suggestion
As more and more projects build on Xarray, we wanted to provide a central place to collect community projects. We hope that including projects under the xarray-contrib organization improves project visibility and increases the likelihood of sustainable open source maintenance and development.
```

## Requirements

> - uses a modern version of Xarray

Reviewer 2: ```suggestion
- Supports a recent version of Xarray
```

# xarray-contrib

xarray-contrib is a GitHub organization for gathering high-quality open source projects that build on [Xarray](http://xarray.pydata.org/).

## Vision

As more and more projects build on Xarray, we wanted to provide a central place to collect community projects. We hope that including projects under the xarray-contrib organization improves project visibility and increases the likelihood of sustainable open source maintenance and development.

## Requirements

- Supports a recent version of Xarray
- Available on GitHub

> - Multiple documented users and/or developers (stars, pull requests, dependencies, etc.)

Reviewer 3: What do you mean by that?
Emerging projects are not welcome?

Reviewer 4: I agree, after thinking a bit more about this I think we should remove this "popularity" requirement to make xarray-contrib welcoming to new project:
1. It is not part of the sklearn-contrib requirements that we based on this idea on.
2. It is already hard enough to bootstrap a new project from scratch! Xarray-contrib should make this easier, rather than being another obstacle.

Reviewer 5: the original intent was to exclude very specific projects which will only ever have a single developer, who is often also the only user. If that makes it more difficult for emerging projects we should indeed remove that requirement (entirely or by moving it to the criteria for reviewing).

Reviewer 4: Nobody writes documentation if they don't hope to have other users :)

Reviewer 5: I would write API documentation for my future self but probably not narrative docs (unless there's something really important)

Reviewer 4: Right, so let's require some form of narrative docs / tutorial -- enough for someone new to get started!

The spirit of the requirements should be "ready to be useful for others," not "already useful" :)

Reviewer 4: I drafted some edits in https://github.com/xarray-contrib/xarray-contrib/pull/2. Please take a look!
